src/schema/firefox-schemas-import.js

```diff
@@ -38,6 +38,8 @@ const schemaRegexes = [
   new RegExp('browser/components/extensions/schemas/.*\\.json'),
```

Comment:
We do not load these and they should not be used for linting (Thunderbird really does not know them).

Reply:
Only later I saw we clone /browser/components/extensions/schemas/pkcs11.json, if this is the reason why the browser folder is listed here, could you add a comment to explain that?

Reply:
Sounds good!